modules/core/src/views/orthographic-view.js

```diff
@@ -21,21 +30,15 @@ export default class OrthographicView extends View {
     // NOTE: automatically calculated from width and height if not provided
     const {
       // projection matrix arguments
-      left = -width / 2, // Left bound of the frustum
-      top = -height / 2, // Top bound of the frustum
+      left = (-width / 2 + offset[0]) * zoom, // Left bound of the frustum
+      top = (-height / 2 + offset[1]) * zoom, // Top bound of the frustum
+      right = (width / 2 + offset[0]) * zoom, // Right bound of the frustum
+      bottom = (height / 2 + offset[1]) * zoom, // Bottom bound of the frustum
 
       near = 1, // Distance of near clipping plane
```

Comment:
@javidhsueh , I would put a breakpoint here and verify `near` and `far` values are same for both layers. If the layers are using different viewState/viewport, verify they are set to same value for both. If they are not same, make them same that might fix the flicker issue in your demos.